Release notes for 8.19.15 (#19043)
elastic/logstash: docs/static/releasenotes.asciidoc

* <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-19-15]]
=== Logstash 8.19.15 Release Notes

==== Updates to dependencies

* Updated Log4j to 2.25.3 https://github.com/elastic/logstash/pull/18943[#18943]

==== Performance improvements

* Improved dead letter queue performance during flush operations https://github.com/elastic/logstash/pull/18874[#18874]

==== Plugins

*Elastic_integration Filter - 8.19.4*

> * Includes `elasticsearch-logging` module https://github.com/elastic/logstash-filter-elastic_integration/pull/441[#441]

Reviewer: ```suggestion
* Sync up with Elasticsearch 8.19 branch to consume required modules for 8.19.14 https://github.com/elastic/logstash-filter-elastic_integration/pull/441[#441]
```

> * Sync up with Elasticsearch 8.19 branch to consume required modules https://github.com/elastic/logstash-filter-elastic_integration/pull/441[#441]

Reviewer 2: ```suggestion
* Sync up dependencies with Elasticsearch `8.19` https://github.com/elastic/logstash-filter-elastic_integration/pull/441[#441]
```

* <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
* <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>

[[logstash-8-19-15]]
=== Logstash 8.19.15 Release Notes

==== Updates to dependencies

* Updated Log4j to 2.25.3 https://github.com/elastic/logstash/pull/18943[#18943]

==== Performance improvements

* Improved dead letter queue performance during flush operations https://github.com/elastic/logstash/pull/18874[#18874]

==== Plugins

*Elastic_integration Filter - 8.19.4*

> * Includes `elasticsearch-logging` module https://github.com/elastic/logstash-filter-elastic_integration/pull/441[#441]

Reviewer 3: While this is what had to be done in order to _use_ the latest 8.19, this plugin release was about re-syncing with Elasticsearch 8.19 branch.

@mashhur do we have clarity about what version of Elasticsearch specifically is embedded in this plugin?

Reviewer: It was just before 8.19.14 release, we can say pre 8.19.14 sync up and on this day we had a BC.
What do you think if we either abstract:
- Sync up with Elasticsearch 8.19 branch to consume required modules 
- or be more specific by adding ".. required modules from 8.19.14"
?

*Kafka Integration - 11.8.8*

> * Fix `sasl_jaas_config` output configuration https://github.com/logstash-plugins/logstash-integration-kafka/pull/247[#247]

Reviewer 3: ```suggestion
* Fix a regression introduced in 11.8.5 that prevented the Kafka Output plugin from being configured with `sasl_jaas_config` https://github.com/logstash-plugins/logstash-integration-kafka/pull/247[#247]
```

*Elasticsearch Output - 11.22.15*

* Improves the logging experience when DLQ used https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1258[#1258]

* Fix: replace deprecated `File.exists?` with `File.exist?` for Ruby 3.4 (JRuby 10) compatibility https://github.com/logstash-plugins/logstash-output-elasticsearch/pull/1238[#1238]

> 

Reviewer 3: This isn't relevant to Logstash 8.x users.

```suggestion
```
